lib/swift/Sources/LinuxHelper.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  LinuxHelper.swift
+//  Thrift
+//
+//  Created by Christopher Simpson on 8/22/16.
+//
+//
+
+import Foundation
+import CoreFoundation
+
+#if os(Linux)
+/// Extensions for Linux for incomplete Foundation API's.
+/// swift-corelibs-foundation is not yet 1:1 with OSX/iOS Foundation
+
+extension URLSession {
+  // Current one uses NSURLRequest which doesn't currently bridge
+  @discardableResult
+  open func dataTask(with request: URLRequest, completionHandler: @escaping (Data?, URLResponse?, Error?) -> Void) -> URLSessionDataTask {
```

Comment:
I’m not too sure how the tagging/branching works in the Swift repo, but I see that the [commit](https://github.com/apple/swift-corelibs-foundation/commit/cffa65a1f933ddb29b7d3d21cd1756caeddc238f) is in `swift-3.0-branch` (as well as `master`) and is included in the `swift-3.0-GM-CANDIDATE` tag.